compiler/rustc_attr/src/builtin.rs

```diff
@@ -1022,14 +1022,19 @@ pub fn find_transparency(
 
 pub fn allow_internal_unstable<'a>(
     sess: &'a Session,
-    attrs: &[Attribute],
+    attrs: &'a [Attribute],
 ) -> Option<impl Iterator<Item = Symbol> + 'a> {
-    let attr = sess.find_by_name(attrs, sym::allow_internal_unstable)?;
-    let list = attr.meta_item_list().or_else(|| {
-        sess.diagnostic()
-            .span_err(attr.span, "allow_internal_unstable expects list of feature names");
-        None
-    })?;
+    let attrs = sess.filter_by_name(attrs, sym::allow_internal_unstable);
+    let list = attrs
+        .filter_map(move |attr| {
+            attr.meta_item_list().or_else(|| {
+                sess.diagnostic()
+                    .span_err(attr.span, "allow_internal_unstable expects list of feature names");
```

Comment:
```suggestion
                    .span_err(attr.span, "`allow_internal_unstable` expects a list of feature names");
```

Reply:
In my defense, I haven't touched that string :)